pgtap/withPoints/withPointsDD/edge_cases/undirected_driving_side.pg

```diff
@@ -58,75 +53,18 @@ PREPARE q3 AS
 SELECT seq, node, edge, cost::TEXT, agg_cost::TEXT FROM pgr_withPointsDD(
     'SELECT id, source, target, cost, reverse_cost FROM edge_table ORDER BY id',
     'SELECT pid, edge_id, fraction, side from pointsOfInterest',
-    -1, 4.8, 'l', details := true, directed:=false)
+    -1, 4.8, driving_side := 'l', details := true, directed:=false)
 ORDER BY seq;
 
-RETURN QUERY
+
 SELECT set_eq('q1', 'q2',
     'Should be equal: both driving sides with right driving side');
 
-RETURN QUERY
 SELECT set_eq('q1', 'q3',
     'Should be equal: both driving sides with left driving side');
 
-RETURN QUERY
 SELECT set_eq('q2', 'q3',
     'Should be equal: right driving sides with left driving side');
 
-END IF;
-
-
-/* TODO remove tests on v4 of the old signatures */
```

Comment:
The PR you are going to make to main repo is for V3.6
So you  can not remove this test, they will be removed on Version V4.